flash/tabular/forecasting/cli.py

```diff
@@ -32,7 +32,9 @@ def from_synthetic_ar_data(
     n_series: int = 100,
     max_encoder_length: int = 60,
     max_prediction_length: int = 20,
-    **data_module_kwargs,
+    batch_size: int = 4,
+    num_workers: int = 0,
+    **time_series_dataset_kwargs,
```

Comment:
so before it was not possible to use `batch_size` or why this sudden separation between `**time_series_dataset_kwargs` and `**data_module_kwargs`

Reply:
Tabular forecasting currently has issues on Flash Zero. I think we will need to switch `time_series_kwargs` to be a dict. At the moment (without this fix) you would get an error where `DataModule` arguments are passed to the `TimeSeriesDataset`, which isn't good haha. So currently the limitation is that you can't pass time series dataset kwargs via flash Zero.